src/components/Heading/Heading.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { variant } from 'styled-system';
+import PropTypes from 'prop-types';
+import theme from '../../theme';
+import { get } from '../../utils/utils';
+import { TYPOGRAPHY, COMMON } from '../../utils/constants';
+
+const style = css`color: ${get('colors.text.coolTone.main')};
```

Comment:
woah cool

Reply:
i deleted it but i guess it's useful for later :')